xla/myplugin/myplugin_cpp_pjrt.h

```diff
@@ -0,0 +1,160 @@
+#ifndef MYPLUGIN_PJRT_CPP_H
+#define MYPLUGIN_PJRT_CPP_H
+
+#include <vector>
+
+#include "xla/pjrt/pjrt_client.h"
+
+namespace myplugin_pjrt {
+  using namespace xla;
+
+  class MypluginPjrtClient : public xla::PjRtClient {
+    public:
+      MypluginPjrtClient() {};
+      ~MypluginPjrtClient() override {};
+      absl::string_view platform_name() const override;
+      int process_index() const override;
+      int device_count() const override { return 42; }
+      int addressable_device_count() const override { return 43; }
```

Comment:
Is there anywhere which documents what these are? If not.. maybe code comments?

Also is the goal of this to serve as another PJRT plugin example or was it just to dogfood docs? No comments needed if just for doc dogfood

Reply:
These are all documented in pjrt_client.h - https://github.com/openxla/xla/blob/main/xla/pjrt/pjrt_client.h#L512

The goal is just to dogfood the docs and ensure the instructions worked.